# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentChild.h
# Commit: 8973242af52a
# Full Hash: 8973242af52a57c4c3471d4e1754953dcbc1c39f
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2025-03-18 21:32:34
# Regressor Bug: 1899503
# File Overlap Count: 2
# Description:
#   Bug 1899503 - P2 PRemoteWorkerDebuggerMangaer binding implementation. r=asuth
#   
#   This patch implements the PRemoteWorkerDebuggerManager binding.
#   
#   PRemoteWorkerDebuggerManager is the IPC that handles the registration/unregistration of remote debuggers.
# ==============================================================================

diff -r 2596059bf325 -r 8973242af52a dom/ipc/ContentChild.h
--- a/dom/ipc/ContentChild.h	Tue Mar 18 12:10:53 2025 +0000
+++ b/dom/ipc/ContentChild.h	Tue Mar 18 12:10:53 2025 +0000
@@ -372,7 +372,8 @@
   const nsACString& GetRemoteType() const override;
 
   mozilla::ipc::IPCResult RecvInitRemoteWorkerService(
-      Endpoint<PRemoteWorkerServiceChild>&& aEndpoint);
+      Endpoint<PRemoteWorkerServiceChild>&& aEndpoint,
+      Endpoint<PRemoteWorkerDebuggerManagerChild>&& aDebuggerChildEp);
 
   mozilla::ipc::IPCResult RecvInitBlobURLs(
       nsTArray<BlobURLRegistrationData>&& aRegistations);